client/rest-high-level/src/main/java/org/elasticsearch/client/ml/dataframe/OutlierDetection.java

```diff
@@ -36,40 +35,42 @@
 public class OutlierDetection implements DataFrameAnalysis {
 
     public static OutlierDetection fromXContent(XContentParser parser) {
-        return PARSER.apply(parser, null);
+        return PARSER.apply(parser, null).build();
+    }
+
+    public static OutlierDetection getDefaultInstance() {
```

Comment:
rename to just `default`? `getDefaultInstance` could imply the same instance is always returned, which is not the case.

Reply:
Renamed to "createDefault" as "default" is a keyword in Java.